### DISCUSSION TITLE
Rejected picture with student ID because of new University name

### DISCUSSION THREAD

**Discussion by Fr0nty** [community/community#136558] [GitHub Education]
### Select Topic Area

Question

### Body

My University changed its name last year from UPB to UNSTPB and when I sent a picture with my student ID that contains the new name it says that there is no university name on the ID. That is not the case as it appears that the name on github has not been updated to the newest version. Does anyone know how should I proceed?

**Comment by davevad93** [comment]
Hi @Fr0nty ,

Try contacting education support by opening a ticket [here](https://support.github.com/contact/education), select `My selected school might have incorrect or incomplete information e.g. domains or campus location`, and describe the issue regarding the name of your university.

**Comment by danialtavakoli** [comment]
It sounds like GitHub's verification system hasn't updated to recognize your university's new name. The best course of action is to contact GitHub Support directly. When you reach out, explain the situation, mentioning the university's name change and providing your student ID with the new name. You can also include any official documentation or a link to your university's website that confirms the name change. This should help GitHub update their records and verify your student status.
